lib/security.js

```diff
@@ -14,7 +14,7 @@ class PDFSecurity {
       if (!info.hasOwnProperty(key)) {
         continue;
       }
-      infoStr += `${key}: ${info[key].toString()}\n`;
+      infoStr += `${key}: ${String(info[key])}\n`;
```

Comment:
```suggestion
      infoStr += `${key}: ${info[key]}\n`;
```

Doing the above will get the same result

Reply:
True words